### PR TITLE
android release 7.6.2

### PR DESCRIPTION
**Pull request by ndesai-newrelic**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2024-12-06'
+version: 7.6.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.6.2.zip'
+---
+
+## New Features
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:8]: New Feature > New

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Implemented OpenTelemetry (Otel) Distributed Tracing.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:10]: - Implements OpenTelemetry (Otel) Distributed Tracing.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Added support for Gzip compression for log data ingestion.
+- Included ANR (Application Not Responding) thread data in the reporting process.
+
+## Bug Fixes
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:14]: Fixed issues

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Resolved an issue with log instrumentation that caused build failures.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:16]: - Fixed an issue with log instrumentation that used to cause build failures.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- Fixed a problem where throwables generated repetitive logs in logcat.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:17]: Please check the technical context of this re-written line:
- Fixed an issue where throwable errors generated repetitive entries in Logcat.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+- Addressed a crash caused by calling `removeAttributes` before the agent was started.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:18]: - Fixed a crash caused by calling `removeAttributes` before starting an agent.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+- Corrected various user-reported issues related to log functionality.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:19]: - fixed user-reported issues with log functionality.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+## Support Statement
+
+We recommend updating the agent at least every three months. Specific policies and timelines regarding the support of Android agents can be found in the [Mobile Monitoring Agents EOL Policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:23]: We recommend updating the agent at least every three months. The specific policies and dates for Android agent support can be found in the [Mobile Monitoring Agents EOL Policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-146.mdx [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
 subject: React Native agent
 releaseDate: '2024-09-16'
-version: 1.4.5
+version: 1.4.6
 downloadLink: 'https://www.npmjs.com/package/newrelic-react-native-agent/v/1.4.5'
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-146.mdx:5]: Please check the link for v1.4.5
It should be:
downloadLink: 'https://www.npmjs.com/package/newrelic-react-native-agent/v/1.4.6'

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx [hunk 10]
```diff
@@ -0,0 +1,11 @@
+---
+subject: React Native agent
+releaseDate: '2024-11-15'
+version: 1.4.6
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx:4]: 1.4.6 > 1.4.7

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+downloadLink: 'https://www.npmjs.com/package/newrelic-react-native-agent/v/1.4.7'
+---
+
+## Improvements
+
+- Added support for Expo 52
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx:10]: - Adds support for Expo 52

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+- Disabled default native C++ crash reporting
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx:11]: - Disables default native C++ crash reporting

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+releaseDate: '2024-12-06'
+version: 7.6.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.6.2.zip'
+---
+
+## New Features
+
+- Implemented OpenTelemetry (Otel) Distributed Tracing.
+- Added support for Gzip compression for log data ingestion.
+- Included ANR (Application Not Responding) thread data in the reporting process.
```
Comment (sujitnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx:12]: - Included Application Not Responding (ANR) thread data in the reporting process.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx (hunk 6 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx (hunk 5 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx (hunk 7 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx (hunk 8 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-146.mdx (hunk 9 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-146.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx (hunk 10 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-147.mdx]: looks good

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx (hunk 13 above)
Comment (ndesai-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx]: looks good

**Review by sujitnewrelic — DISMISSED** [review]
LGTM

**Review by sujitnewrelic — APPROVED** [review]
LGTM

**Pull request merged**